test/Microsoft.AspNetCore.Razor.Design.Test/project.json

```diff
@@ -16,16 +16,11 @@
           "version": "1.1.0-*",
           "type": "build"
         },
-        "System.Diagnostics.Process": "4.1.0-*",
-        "xunit": "2.1.0"
-      },
-      "imports": [
-        "dnxcore50",
-        "portable-net451+win8"
-      ]
+        "xunit": "2.2.0-*"
+      }
     }
   },
   "dependencies": {
-    "dotnet-test-xunit": "1.0.0-*"
+    "dotnet-test-xunit": "2.2.0-*"
```

Comment:
Could you move the rest of the dependencies to this node? Doesn't look like we cross compile this project.